feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -94,6 +101,7 @@ class BillingWrapperTest {
 
     @Before
     fun setup() {
+        clearAllMocks()
```

Comment:
Since we are using `mockkStatic` in some of these tests, this was necessary to make tests pass.